pkg/policy/compiler.go

```diff
@@ -25,6 +25,11 @@ type Compiler interface {
 	Compile(*v1alpha1.AuthorizationPolicy) (PolicyFunc, field.ErrorList)
 }
 
+type AuthorizationPrograms struct {
```

Comment:
This doesn't need to be public, you can move the declaration inside the compilation func.